src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/management/MenuTestsIT.java

```diff
@@ -35,11 +35,12 @@ public void beforeTest() throws InterruptedException {
       .mockOrganizations("0de268cf-1ea1-4719-8a6e-1150933b6b9e");
     
     getManagementPageMocker()
-      .mockPages(456, 567, 678)
-      .startMock();
+      .mockPages(456, 567, 678);
     
+    getManagementMediaMocker()
+      .mockMedias(3001, 3002);;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)